x/incentive/keeper/store.go

```diff
@@ -0,0 +1,275 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/umee-network/umee/v4/util/store"
+	"github.com/umee-network/umee/v4/x/incentive"
+)
+
+// GetMaxUnbondings gets the maximum number of unbondings an account is allowed to have at one time.
+func (k Keeper) GetMaxUnbondings(ctx sdk.Context) uint32 {
+	return store.GetUint32(k.KVStore(ctx),
+		keyPrefixParamMaxUnbondings, 0, "max unbondings")
+}
+
+// GetUnbondingDurationLong gets the duration in seconds of the long bonding tier.
+func (k Keeper) GetUnbondingDurationLong(ctx sdk.Context) uint64 {
+	return store.GetUint64(k.KVStore(ctx),
+		keyPrefixParamUnbondingDurationLong, 0, "long unbonding duration")
+}
+
+// GetUnbondingDurationMiddle gets the duration in seconds of the middle bonding tier.
+func (k Keeper) GetUnbondingDurationMiddle(ctx sdk.Context) uint64 {
+	return store.GetUint64(k.KVStore(ctx),
+		keyPrefixParamUnbondingDurationMiddle, 0, "middle unbonding duration")
+}
+
+// GetUnbondingDurationShort gets the duration in seconds of the short bonding tier.
+func (k Keeper) GetUnbondingDurationShort(ctx sdk.Context) uint64 {
+	return store.GetUint64(k.KVStore(ctx),
+		keyPrefixParamUnbondingDurationShort, 0, "short unbonding duration")
+}
+
+// GetTierWeightShort gets the ratio of rewards received by the short tier of bonded assets. Ranges 0 - 1.
+func (k Keeper) GetTierWeightMiddle(ctx sdk.Context) sdk.Dec {
+	return store.GetDec(k.KVStore(ctx),
+		keyPrefixParamTierWeightMiddle, sdk.ZeroDec(), "middle tier weight")
+}
+
+// GetTierWeightShort gets the ratio of rewards received by the middle tier of bonded assets. Ranges 0 - 1.
+func (k Keeper) GetTierWeightShort(ctx sdk.Context) sdk.Dec {
+	return store.GetDec(k.KVStore(ctx),
+		keyPrefixParamTierWeightShort, sdk.ZeroDec(), "short tier weight")
+}
```

Comment:
if you will need to read multiple params in a single request, then probably it's better to serialize all params as a single record (Params), rather than breaking it down into multiple records.

Reply:
I'll have to think about this one - I'm thinking
- in most transactions only one param is needed
- in endblocker multiple params will be needed
- in non-iterating queries zero or one are likely to be needed (e.g. unless a person has locked multiple tiers of the same token)

Might keep as-is for now, but I can check up on it after implementation PRs